curationexperts-deprecated/tenejo-v1
# 304 update ruby and rails in test suite

This pull request upgrades Tenejo to Ruby 2.7.4 and Ruby on Rails to v. 5.2.6. This includes changes to numerous files as a result of the change in Rubocop version (see below) and items e.g., .ruby-version, to placate CircleCI. The suite upgrade requires local upgrades to Ruby and bundler.

Other updated dependencies include Sidekiq to 5.2.7 (formerly 5.1.3), bixby to  3.0.2 (2.0.0.pre.beta1), which upgrades rubocop to version 0.85.1 (0.63). 

#### To test:
- install ruby 2.7.4 using your preferred ruby installer
- `gem install bundler:2.2.28`
- `git checkout 304-update_ruby_and_rails_in_test_suite_rubocop_corrected`
-  `bundle install`
- migrate test db
-- `bundle exec rails db:migrate RAILS_ENV=test`
- start tenejo test suite as directed in [tenejo README](https://github.com/curationexperts/tenejo/blob/main/README.md).
- run tests
-- `bundle exec rspec spec/`
-- verify tests run as expected.

Done.
